src/openapi/authentication.ts

```diff
@@ -17,3 +17,13 @@ export const authenticate = (JWT: string): void => {
 };
 
 export const isAuthenticated = (): boolean => !!myWindow?.sessionStorage.getItem('JWT');
+
+export const getBsnFromJWT = (): string => {
+  const JWT = myWindow?.sessionStorage.getItem('JWT');
```

Comment:
For this one it's required because `getBsnFromJWT` is called in server side rendering as well, which doesn't have a sessionStorage and would throw an undefined error.